BEIMA.Client/src/services/AddDevice.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+const API_URL = process.env.REACT_APP_API_URL;
+
+/**
+ * POSTs a JSON object to the API for storage in DB
+ *
+ * @param The device details to add to the DB
+ * @return Error message or the inserted device ID
+ */
+export default async (deviceDetails) => {
```

Comment:
I don't see a name for this function how do we call it?

Reply:
import <whatever you want to call it in your page js> from '../../AddDevice.js'

accessing is <whatever you want to call it in your page js>(<your argument here>)
GetDevice doesn't have a name either

Reply:
We should give it a name so that it's clear what this function does and that we are using the same name consistently throughout our front end

Reply:
Can this change be made to all of the open PRs?

Reply:
sure